src/Core/Analysis/AnalysisLanguage.cs

```diff
@@ -27,6 +27,7 @@ public enum AnalysisLanguage
     {
         Javascript,
         CFamily,
-        RoslynFamily
+        RoslynFamily,
+        TypeScript
```

Comment:
I'll change SonarLanguageRecognizer in the next PR to actually detect the right content type.